tests/haskell/GhcModBearTest.py

```diff
@@ -0,0 +1,96 @@
+from queue import Queue
+
+from bears.haskell.GhcModBear import GhcModBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+# A simple hello world program!
+hello_world_file = """
+main :: IO()
+main = putStrLn "Hello World!"
+"""
+
+# No instance for (Num String) in the first argument of ‘putStrLn’
+not_string_3 = """
+main :: IO()
+main = putStrLn 3
+"""
+
+# Takes a file and prints the number of lines
+count_lines = """
+main :: IO()
+main = interact lineC
+    where lineC input = show (length(lines input))
+"""
+
+# Parse Error (mismatched brackets)
+count_lines_bracket_missing = """
+main :: IO()
+main = interact lineC
+    where lineC input = show (length(lines input)
+"""
+
+# Prints "asd" which is the last but one element in the list
+last_but_one_list = """
+lastButOne :: [a] -> a
+lastButOne xs | (length xs >= 2) = xs !! ((length xs) - 2)
+              | otherwise = error "Not possible"
+
+main :: IO()
+main = do
+    print(lastButOne(["asd", "ASd"]))
+"""
+
+# Couldn't match expected type ‘[a]’ with actual type ‘a’
+last_but_one_list_type_error = """
+lastButOne :: a -> a
+lastButOne xs | (length xs >= 2) = xs !! ((length xs) - 2)
+              | otherwise = error "Not possible"
+
+main :: IO()
+main = do
+    print(lastButOne(["asd", "ASd"]))
+"""
+
+
+@generate_skip_decorator(GhcModBear)
+class GhcModBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('name')
+        self.uut = GhcModBear(self.section, Queue())
+
+    def test_valid(self):
+        self.check_validity(self.uut, hello_world_file.split('\n'), valid=True,
```

Comment:
actually you could do that already for your strings instead of doing it here :+1: